### PR TITLE
frontend: also support vertical in fullscreen mode

### PR DESCRIPTION
**Pull request by thisconnect**
Trying to make verticallyCentered also work for fullscreen

This still needs more testing and is not ready

- should center all text content in the view (fullscreen and fill should work
- fixes undefined in classname
- fixes paddings with and without sidebar on small, medium and large screen

views to test:

- alertUser: settings -> fullnode -> test with invalid url
- bb02 device settings
- enable passphrase
- address request

### CODE REVIEW

File: frontends/web/src/components/view/view.module.css [hunk 1]
```diff
@@ -38,16 +38,19 @@
...
+    margin: 0 auto;
+    max-width: var(--content-width);
+    padding: var(--space-half) 0;
+    width: 100%;
 }
-.fill:not(.verticallyCentered) .inner {
-    margin: 0;
+/* .fill .inner {
     width: 100%;
+} */
```
Comment (thisconnect) [inline frontends/web/src/components/view/view.module.css]: need to cleanup

**Comment by shonsirsha** [comment]
> This still needs more testing and is not ready


Is it still the case or is it ready for review now? :)

EDIT: Seems like development process is still ongoing. Feel free to ping me here once it's ready for review. Thx!

**Review by shonsirsha — APPROVED** [review]
tested LGTM for the following components:

alertUser: settings -> fullnode -> test with invalid url
bb02 device settings
enable passphrase
AOPP

**Comment by thisconnect** [comment]
yeah sorry, I forgot to remove the comment. When I first opened it was the case, but then I found some issue and force pushed a fix.

> This still needs more testing and is not ready

**Comment by thisconnect** [comment]
@shonsirsha should I remove my test-commit or do you want to cherry-pick the other commit and squash into your commit?

**Comment by shonsirsha** [comment]
>  @shonsirsha should I remove my test-commit or do you want to cherry-pick the other commit and squash into your commit?

@thisconnect I think you could remove the test commit and then merge this one (into [my branch](https://github.com/shonsirsha/bitbox-wallet-app/tree/frontend-modify-view-component)). Then we can continue in the upstream PR https://github.com/digitalbitbox/bitbox-wallet-app/pull/1935 . What do you think?

**Comment by thisconnect** [comment]
I removed my commit, but I think you should squash it into yours

**Comment by shonsirsha** [comment]
Closing this as it's been squashed into https://github.com/digitalbitbox/bitbox-wallet-app/pull/1935. Thanks @thisconnect !

**Pull request closed without merging**